test/fixtures/dialects/snowflake/snowflake_alter_table_column.yml

```diff
@@ -613,14 +642,14 @@ file:
       - naked_identifier: my_table
     - alter_table_table_column_action:
         keyword: add
-        column_definition:
+        column_reference:
```

Comment:
does this matter?

Reply:
Yes it does. It now thinks that `PRIMARY` is a column name, rather than a column definition, meaning it will be subject to the identifier rules, rather than the keyword rules - for example rules L010 and L014.

Though it looks to me like it didn't parse this part properly anyway since it thinks `PRIMARY` is an identifier and `KEY` is a data type. Still I think we've made it worse with this change.